<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cavland Hostel Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_body">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1051</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="nav_bar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>251</width>
       <height>601</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 26, 81);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CAVLAND HOSTEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnDashboard">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(27, 43, 101);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Dashboard</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnRoom">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Rooms</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnStudent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Students</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAnalytics">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Analytics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnExpansion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Expansions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnStaff">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Staff</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnReport">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Reports</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnLog">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(8, 26, 81);
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
text-align: left;
padding-left: 60px;</string>
      </property>
      <property name="text">
       <string>Logs</string>
      </property>
     </widget>
     <zorder>Title</zorder>
     <zorder>BtnRoom</zorder>
     <zorder>BtnStudent</zorder>
     <zorder>BtnAnalytics</zorder>
     <zorder>BtnExpansion</zorder>
     <zorder>BtnDashboard</zorder>
     <zorder>BtnStaff</zorder>
     <zorder>BtnReport</zorder>
     <zorder>BtnLog</zorder>
    </widget>
    <widget class="QFrame" name="main_contemt">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>801</width>
       <height>541</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>-3</x>
        <y>-10</y>
        <width>801</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="DashobardArea">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Dashboard</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>520</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Database Connection</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="RoomsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Rooms</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="StudentsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Students</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="AnalyticsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Analytics</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="ExpansionsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Expansions</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="ReportsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Reports</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="StaffArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Staff</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="LogsArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>441</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>31</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Logs</string>
       </property>
      </widget>
     </widget>
     <zorder>line</zorder>
     <zorder>RoomsArea</zorder>
     <zorder>StudentsArea</zorder>
     <zorder>AnalyticsArea</zorder>
     <zorder>ExpansionsArea</zorder>
     <zorder>ReportsArea</zorder>
     <zorder>StaffArea</zorder>
     <zorder>LogsArea</zorder>
     <zorder>DashobardArea</zorder>
    </widget>
    <widget class="QFrame" name="main_header">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>801</width>
       <height>61</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>1600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="LblTitle">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Medium</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Dashboard</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
